targets/TARGET_STM/can_api.c

```diff
@@ -466,7 +466,7 @@ void CAN_IRQHandler(void)
 }
 #endif
 
-#if defined(TARGET_STM32F1) || defined(TARGET_STM32F2)
+#if defined(TARGET_STM32F1) || defined(TARGET_STM32F2) || defined(TARGET_STM32F4)
```

Comment:
could we use 
#if defined(TARGET_STM32F0)
..
#else

rather than the family list ? 
Or maybe a specific property name that can be defined in can_device.h files ? 
In case a new family is added, this would not require a change here.